deploy/bundle.yaml

```diff
@@ -56,15 +56,14 @@ spec:
                 description: Region is a region where the bucket is located.
                 type: string
               type:
-                description: Type is a type of backup storage. Currently only S3 is supported.
+                description: Type is a type of backup storage.
                 enum:
                 - s3
+                - azure
                 type: string
             required:
             - bucket
             - credentialsSecretName
-            - endpointURL
-            - region
```

Comment:
I assume this is a leftover from the Azure support changes, the bundle wasn't rebuild back then.

Reply:
Yes that's it.